src/realtime.c

```diff
@@ -256,9 +263,9 @@ load_all_properties (GDBusProxy *proxy)
       GVariant *parameters;
       g_autoptr (GError) error = NULL;
 
-      parameters = g_variant_new ("(ss)", "org.freedesktop.RealtimeKit1", properties[i]);
+      parameters = g_variant_new ("(ss)", RTKIT_DBUS_IFACE, properties[i]);
       result = g_dbus_proxy_call_sync (proxy,
-                                       "org.freedesktop.DBus.Properties.Get",
+                                       DBUS_DBUS_IFACE ".Properties.Get",
                                         g_steal_pointer (&parameters),
```

Comment:
Spacing issue, but overall I think the bigger question is why the code works this way and doesn't just use the cached properties already loaded by the proxy object? (i.e. bind the impl proxy's properties to the Realtime object and call it a day?)

There would be one difference in behavior: the code as-written ignores property change events, but binding the properties wouldn't do that. But I have a sneaking suspicion that these properties never actually change